src/utils/daos.c

```diff
@@ -1066,6 +1069,26 @@ fs_op_hdlr(struct cmd_args_s *ap)
 			ARGS_VERIFY_PUUID(ap, out, rc = RC_PRINT_HELP);
 			ARGS_VERIFY_CUUID(ap, out, rc = RC_PRINT_HELP);
 		}
+
+		rc = daos_pool_connect(ap->p_uuid, ap->sysname, DAOS_PC_RW,
+				&ap->pool, NULL, NULL);
+		if (rc != 0) {
+			fprintf(stderr,
+				"failed to connect to pool "DF_UUIDF": %s (%d)\n",
+				DP_UUID(ap->p_uuid), d_errdesc(rc), rc);
+			return rc;
+		}
+
+		rc = daos_cont_open(ap->pool, ap->c_uuid,
+				    DAOS_COO_RW | DAOS_COO_FORCE,
+				    &ap->cont, NULL, NULL);
+		if (rc != 0) {
+			fprintf(stderr,
+				"failed to open container "DF_UUIDF ": %s (%d)\n",
```

Comment:
(style)  line over 80 characters

Reply:
Whitespace.  I don't know if it's enough to solve the line length though.
```suggestion
				"failed to open container "DF_UUIDF": %s (%d)\n",
```